tests/e2e/tests/test_sanity.py

```diff
@@ -224,3 +244,30 @@ def test_katib_experiment(self, setup, cluster, region):
             raise AssertionError("Expected K8sApiException Not Found")
         except K8sApiException as e:
             assert "Not Found" == e.reason
+
+    def test_kserve_with_irsa(self, region, metadata, kfp_client, clone_tensorflow_serving, kserve_iam_service_account, kserve_secret, s3_bucket_with_training_data, kserve_inference_service):
```

Comment:
if you inject dependency in kserve_inference_service  fixture, you would not need to add all of them here, just `kserve_inference_service` see how load_balancer is handled